pkg/option/config.go

```diff
@@ -3954,3 +3972,67 @@ func MightAutoDetectDevices() bool {
 		(Config.KubeProxyReplacement != KubeProxyReplacementDisabled &&
 			(len(devices) == 0 || Config.DirectRoutingDevice == ""))
 }
+
+// BPFEventBufferConfig contains parsed configuration for a bpf map event buffer.
+type BPFEventBufferConfig struct {
+	Enabled bool
+	MaxSize int
+	TTL     time.Duration
+}
+
+// BPFEventBufferConfigs contains parsed bpf event buffer configs, indexed but map name.
+type BPFEventBufferConfigs map[string]BPFEventBufferConfig
+
+// GetEventBufferConfig returns either the relevant config for a map name, or a default
+// one with enabled=false otherwise.
+func (d *DaemonConfig) GetEventBufferConfig(name string) BPFEventBufferConfig {
+	return d.bpfMapEventConfigs.get(name)
+}
+
+func (cs BPFEventBufferConfigs) get(name string) BPFEventBufferConfig {
+	return cs[name]
+}
+
+// ParseVentBufferTupleString parses a event buffer configuration tuple string.
+// For example: true,100,24h
+// Which refers to enabled=true, maxSize=100, ttl=24hours.
```

Comment:
`enabled` here seems redundant (or confusing). I see a few cases
1. The map is disabled by default, so I'd like to enable it and tweak the parameters. Then just `mapname=100,24`
2. The map is disabled by default, and we'd like to enable it with default size. Then just `mapname,mapname1`
3. The map is enabled by default, and we'd like to disable it. Them `mapname=disabled` would be what I would write, not `mapname=false,0,0` or something.

Reply:
Did some work on this, I think I prefer consistency for the tuple structure however 👍 on the true->enabled change.